docs/api/bluetooth/GattClient.md

```diff
@@ -16,6 +16,14 @@ As a result of the discovery process, the client can start interacting with the
 
 Mbed BLE abstracts read and write operations to offer a single API that can be used to read or write characteristic values. The application code does not have to handle the necessary fragmentation or reassembly process if the attribute value to be transported can't fit in a single data packet.
 
+#### ATT_MTU
```

Comment:
Maybe we can rework the section name into something less cryptic. What do you think of _GATT Maximum Transmission Unit (MTU)_ ?

Reply:
how about Attribute Maximum Transmission Unit (ATT_MTU)

Reply:
I'm not sure: we do not even define what GATT stands for in this file; `Attribute` seems a bit alien to me given the context.

Reply:
But it is technically correct. The packet is an Attribute Protocol packet. I'm taking the terms from the spec to be consistent with other literature.

Reply:
I do not deny that. I'm just pointing that it would be good to introduce the term Attribute Protocol (ATT) and Generic Attribute Profile (GATT) before the title. But we can go with _Attribute Protocol Maximum Transmission Unit_

Reply:
I introduced the acronyms and the explanation of their relationship.